src/__tests__/components/SearchForm.test.js

```diff
@@ -0,0 +1,47 @@
+import React, { fireEvent, render, screen } from "@testing-library/react";
+// import renderer from "react-test-renderer";
+import SearchForm from "../../components/SearchForm";
+
+describe("SearchForm", () => {
+  const validProps = {
+    searchText: "Manchester",
+    setSearchText: jest.fn(),
+    onSubmit: jest.fn(),
+  };
+
+    it("renders correctly", () => {
+        const { asFragment } = render(
+          <SearchForm 
+             searchText={validProps.searchText} 
+             setSearchText={validProps.setSearchText}
+             onSubmit={validProps.onSubmit}
+          />
+        );
+        expect(asFragment()).toMatchSnapshot();
+    });
+
+    it("button displays the correct text", () => {
+        render(
+          <SearchForm 
+            searchText={validProps.searchText} 
+            setSearchText={validProps.setSearchText}
+            onSubmit={validProps.onSubmit}
+          />
+        );
+        const button = screen.getAllByRole("button");
+
+        expect(button[0]).toHaveClass("search__button")
```

Comment:
Are you expecting to only have one button? If so, would be good to add another assertion to check the length of the buttons returned by the `screen.getAllByRole` query